### DISCUSSION TITLE
New Source: Ironsource

### DISCUSSION THREAD

**Discussion by d1bevz** [airbytehq/airbyte#34299] [New Connector Request]
### Topic

connector request

### Relevant information

Is there any chance that Ironsource will be added?

**Comment by marcosmarxm** [comment]
New request must be created as discussion. Move to discussions.

**Reply by d1bevz** [reply]
Sorry, I didn't know

**Comment by d1bevz** [comment]
I mean data like this: https://developers.is.com/ironsource-mobile/air/reporting/

**Comment by marcosmarxm** [comment]
@d1bevz did you try to use the connector builder UI to help you?

**Reply by d1bevz** [reply]
Could you please send me the link to docs? I'm new to Airbyte technology, I am not sure what is it

**Reply by d1bevz** [reply]
@marcosmarxm I figured out how to use builder UI, but it seems like Ironsource uses [tricky Bearer token for authentification](https://developers.is.com/ironsource-mobile/air/authentication/#step-1) which works only for 60 minutes, so it won't be possible to automate using builder

[a comment by an automated account is not shown]
